packages/bierzo-wallet/src/routes/payment/components/TextNote/index.tsx

```diff
@@ -10,34 +10,34 @@ interface Props {
 }
 
 const useStyles = makeStyles(() => ({
-  paperRoot: {
+  paper: {
```

Comment:
Can you remind me the usage of `positionedClass`?

Remember the rule for styles, fontSizes should be declared using Typography and its variant prop, not directly the size, in this way, if we change the Typography component, everything will be updated accordingly.

And the margin and padding should be specified in Block component via props, not css.